aiidalab_widgets_base/computational_resources.py

```diff
@@ -927,11 +927,20 @@ def on_setup_computer_success(self, function):
     def test(self, _=None):
         with self._test_out:
             clear_output()
-            print(
-                subprocess.check_output(
-                    ["verdi", "computer", "test", "--print-traceback", self.label.value]
-                ).decode("utf-8")
-            )
+            try:
+                print(
+                    subprocess.check_output(
```

Comment:
I wonder if we can use some other function (POpen?) instead of this, because when the verdi commands fails, the user will currently only get the exit code, but not the error message from verdi which is not very helpful.

![obrazek](https://github.com/aiidalab/aiidalab-widgets-base/assets/9539441/c4b733cc-0ec6-42f2-8260-1b1e1060f516)